src/Middleware/ResponseCaching/src/ResponseCachingExtensions.cs

```diff
@@ -6,8 +6,15 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for the ResponseCachingMiddleware middleware.
```

Comment:
```suggestion
    /// Extension methods for adding the <see cref="ResponseCachingMiddleware"/> to an application.
```

Consistency with [other places](https://github.com/dotnet/aspnetcore/blob/87629bbad906e9507026692904b6bcb5021cdd33/src/Middleware/Session/src/SessionMiddlewareExtensions.cs#L11).